jans-cedarling/cedarling/src/jwt/validation.rs

```diff
@@ -3,408 +3,8 @@
 //
 // Copyright (c) 2024, Gluu, Inc.
 
-use std::collections::HashSet;
+mod validator;
+mod validator_cache;
 
```

Comment:
status list validation errors were mixed with general JWT validation errors?
better to separate them and define status list validation errors in this file so that it can improve error handling and debugging?